language-support/java/codegen/src/main/scala/com/digitalasset/daml/lf/codegen/InterfaceTree.scala

```diff
@@ -215,11 +214,3 @@ private[codegen] object InterfaceTree extends StrictLogging {
     }
   }
 }
-
-private[codegen] final case class InterfaceTrees(interfaceTrees: List[InterfaceTree])
-
-private[codegen] object InterfaceTrees extends StrictLogging {
-
-  def fromInterfaces(interfaces: Seq[Interface]): InterfaceTrees =
-    InterfaceTrees(interfaces.view.map(InterfaceTree.fromInterface).toList)
-}
```

Comment:
Not a particularly useful wrapper, removed and replaces with a simple sequence of `Interface`s.